packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItemModal.jsx

```diff
@@ -560,7 +606,7 @@ const DataSeriesFormItemModal = ({
             </div>
           )}
 
-        {(type === CARD_TYPES.VALUE || type === CARD_TYPES.IMAGE || type === CARD_TYPES.TABLE) && (
+        {hasThresholds && (
```

Comment:
Here perhaps the old logic should be kept in case `hasThreshold` hasn't been provided, also for backward compatibility.

Reply:
same comment as https://github.com/carbon-design-system/carbon-addons-iot-react/pull/3871#discussion_r1650967130